frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -216,7 +216,20 @@ const TeamEventCreditForm: React.FC = () => {
 
         <div style={{ margin: '2rem 0' }}>
           <label style={{ fontWeight: 'bold' }}>Approved Events:</label>
-          <p>{`[${approvedTEC}]`}</p>
+          {approvedTEC.length !== 0 ? (
+            <Card.Group>
+              {approvedTEC.map((teamEvent) => (
+                <Card>
+                  <Card.Content>
+                    <Card.Header>{teamEvent.name} </Card.Header>
+                    <Card.Meta>{teamEvent.date}</Card.Meta>
+                  </Card.Content>
```

Comment:
I think displaying the number of credits would also be good

Reply:
Good idea!